docs/source/API/containers/Array.rst

```diff
@@ -0,0 +1,173 @@
+..
+  Use the following convention for headings:
+
+    # with overline, for parts (collections of chapters)
+
+    * with overline, for chapters
+
+    = for sections
+
+    - for subsections
+
+    ^ for subsubsections
+
+    " for paragraphs
+
+..
+  Class / method / container name)
+  for free functions that are callable, preserve the naming convention, `view_alloc()`
+
+``Array``
+==============
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+..
+  The (pulic header) file the user will include in their code
+
+Header File: ``Kokkos_Core.hpp``
+
+..
+  High-level, human-language summary of what the thing does, and if possible, brief statement about why it exists (2 - 3 sentences, max);
+
+Description
```

Comment:
Since we have the old version below: maybe add in which version this one was added?

Reply:
Done.